### DISCUSSION TITLE
storescp --filepath '{00081030}/{0008103E}' fails for Study/Series Descriptions including characters not allowed in file names

### DISCUSSION THREAD

**Discussion by 00exia00** [dcm4che/dcm4che#859] [Q&A] [answered]
(0008,1030)&(0008,103E)

when I use --filepath <pattern> this options, and set (0008,1030)&(0008,103E)(study description/series description) these two for patterns. Once the description include such as "/ *" the trasmission stops. Because both in windows and mac cannot create a folder use character like  "< > | ? " * : / \ ". 

Can you set once meet the special character then replace to "_" or "-" or any character that OS can use.

**Comment by gunterze** [comment, marked as answer]
Use `--filepath '{00081030,urlencoded}/{0008103E,urlencoded}'`
